Microsoft.Toolkit.Sample.UWP.App/Microsoft.Toolkit.Sample.UWP.App.csproj

```diff
@@ -169,14 +171,16 @@
     </PackageReference>
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\Microsoft.Toolkit.Win32.UI.XamlApplication\Microsoft.Toolkit.Win32.UI.XamlApplication.vcxproj">
+    <Reference Include="..\$(XamlApp-Platform)\$(Configuration)\Microsoft.Toolkit.Win32.UI.XamlApplication\Microsoft.Toolkit.Win32.UI.XamlHost.winmd" />
+    <!--ProjectReference Include="..\Microsoft.Toolkit.Win32.UI.XamlApplication\Microsoft.Toolkit.Win32.UI.XamlApplication.vcxproj">
```

Comment:
Remove

Reply:
Why did you switch from ProjectReference to a Reference? I've switched it back to a ProjectReference which seems better. Is there a reason not to do that?